portfolio/src/main/webapp/javascript/script.js

```diff
@@ -16,9 +17,67 @@ async function getComments() {
   }
 }
 
-/** Creates an <li> element containing text. */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+/* Checks if comments are hidden or not*/
+async function checkComments() {
+  let btn = document.getElementById("comment-btn");
```

Comment:
Try to avoid using abbreviations that are not universally recognized. Changing `btn` -> `button` can help improve the readability of your code.